hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/context/NodeInfoTest.java

```diff
@@ -63,28 +63,28 @@ void setUp() {
     }
 
     @Test
-    void understandsStaked() {
-        givenEntryWithStake(nodeId, 1L);
+    void understandsNonZeroWeight() {
+        givenEntryWithWeight(nodeId, 1L);
 
         // expect:
-        assertFalse(subject.isZeroStake(nodeId));
-        assertFalse(subject.isSelfZeroStake());
+        assertFalse(subject.isZeroWeight(nodeId));
+        assertFalse(subject.isSelfZeroWeight());
     }
 
     @Test
-    void understandsZeroStaked() {
-        givenEntryWithStake(nodeId, 0L);
+    void understandsZeroWeight() {
```

Comment:
same here in this class

Reply:
Done. I updated all variables and methods except for `givenEntryWithWeight` and `givenEntryWithMemoAndWeight`. It made sense to me to leave this as `*Weight` because they are using `address.getWeight()`